huaweicloud/services/dew/resource_huaweicloud_csms_secret.go

```diff
@@ -240,11 +240,11 @@ func resourceCsmsSecretUpdate(ctx context.Context, d *schema.ResourceData, meta
 			KmsKeyID:    kmsKeyID,
 			Description: &desc,
 		}
-		logp.Printf("[DEBUG] The option to update the basic information of the CSMS secret is: %#v", opts)
+		log.Printf("[DEBUG] The option to update the basic information of the CSMS secret is: %#v", opts)
 
 		_, err = secrets.Update(client, name, opts)
 		if err != nil {
-			e := fmtp.Errorf("failed to update the base-info of CSMS secret: %s", err)
+			e := fmt.Errorf("failed to update the base-info of CSMS secret: %s", err)
 			mErr = multierror.Append(mErr, e)
```

Comment:
directly return the error, don't need to use multierror

Reply:
fixed